tests/system/libraries/SystemTestSpy/blockUntilConditionMet.py

```diff
@@ -12,28 +12,39 @@
 from time import perf_counter as _timer
 from typing import Any, Callable, Optional, Tuple
 
+EvaluatorWasMetT = bool
+GetValueResultT = Any
 
 def _blockUntilConditionMet(
-		getValue: Callable[[], Any],
+		getValue: Callable[[], GetValueResultT],
 		giveUpAfterSeconds: float,
 		shouldStopEvaluator=lambda value: bool(value),
-		intervalBetweenSeconds: float = 0.1,
+		intervalBetweenSeconds: float = 0.3,

```

Comment:
Considering this value has been updated across usages, perhaps it should be abstracted to a named constant.

```suggestion
DEFAULT_INTERVAL_BETWEEN_SECONDS = 0.3
...
		intervalBetweenSeconds: float = DEFAULT_INTERVAL_BETWEEN_SECONDS,
```

Reply:
I've added a constant for `DEFAULT_INTERVAL_BETWEEN_EVAL_SECONDS` and for `_MIN_INTERVAL_BETWEEN_EVAL_SECONDS`, anywhere explicitly passing in `0.3` now uses the default constant.